src/pathwayvis/components/map/views/map.component.scss

```diff
@@ -57,4 +57,56 @@ pv-map {
 
 .context-menu {
     width: 100%
-}
\ No newline at end of file
+}
+.container-style {
```

Comment:
All css should go in the element's selector (`pv-map` in this case). I know you didn't introduce this change, but it'd be nice if you could fix it. Could you please check if it's still working if you move this part of the code within 
```
pv-map {
  ...
}
```
?